src/debug/debugUtils.ts

```diff
@@ -57,31 +57,40 @@ export async function promptForAppPort(ports: string[], defaultPort: string, env
  * @param container the container name.
  * @return the commands associated with the processes.
  */
-export async function getCommandsOfProcesses(kubectl: Kubectl, pod: string, podNamespace: string | undefined, container: string): Promise<string[]> {
-    const commandLines: string[] = [];
-    const nsarg = podNamespace ? `--namespace ${podNamespace}` : '';
+export async function getProcesses(kubectl: Kubectl, pod: string, podNamespace: string | undefined, container: string | undefined): Promise<ProcessInfo[] | undefined> {
+    const processes: ProcessInfo[] = [];
+    const nsarg = podNamespace && podNamespace !== "default" ? `--namespace ${podNamespace}` : '';
     const containerCommand = container ? `-c ${container}` : '';
-    const execCmd = `exec ${pod} ${nsarg} ${containerCommand} -- ps -ef`;
+    const execCmd = `exec ${pod} ${nsarg} ${containerCommand} -- ps -o pid,command -e -w -w`;

```

Comment:
There are two occurrences of `-w` here - is this intentional?

Reply:
The second `-w` ensures that all columns are displayed despite terminal width.  I can add a comment here.  Probably doesn't matter for this case, because we aren't outputting to a terminal, but helps someone debugging and who wants to run the same command locally and get the same output that gets parsed.

```
     -w      Use 132 columns to display information, instead of the default which is your window size.  If the -w option is specified more than once, ps will use as many columns as
             necessary without regard for your window size.  When output is not to a terminal, an unlimited number of columns are always used.
```

Reply:
added comment